src/redux/features/tracking/trackingSelectors.ts

```diff
@@ -6,3 +6,7 @@ export const selectPlantingSitesError = (state: RootState) => state.tracking?.er
 
 export const selectPlantingSite = (state: RootState, plantingSiteId: number) =>
   selectPlantingSites(state)?.find((site: PlantingSite) => site.id === plantingSiteId);
+
+export const selectSitePopulation = (state: RootState) => state.sitePopulation?.zones;
+
+export const selectSitePopulationError = (state: RootState) => state.sitePopulation?.error;
```

Comment:
if we expect a lot more selector variants for site populations.. suggest moving this to a separate file.. otherwise this is fine